docs/api/mock.md

```diff
@@ -70,6 +70,8 @@ function mockImplementation(fn: T): MockInstance<T>
 
 Accepts a function to be used as the mock implementation. TypeScript expects the arguments and return type to match those of the original function.
 
+**vi.fn(implementation) is a shorthand for vi.fn().mockImplementation(implementation)**
```

Comment:
This is not strictly true. e.g. when using [mockReset / mockRestore](https://vitest.dev/api/mock.html#mockrestore)

> Note that restoring a mock from vi.fn() will set implementation to an empty function that returns undefined. Restoring a mock from vi.fn(impl) will restore implementation to impl.